buildbot/buildbot/worker.py

```diff
@@ -494,13 +494,13 @@ def stage_fpga_execute(db, config):
                 exe_cmd = ['sudo', 'sh', '-c',
                            'source /opt/xilinx/xrt/setup.sh ; ./host']
             else:
-                exe_cmd = ['cur=`pwd`; cd $AWS_FPGA_REPO_DIR ;\
+                exe_cmd = ['sh','-c','cur=`pwd`; cd $AWS_FPGA_REPO_DIR ;\
```

Comment:
Typical Python style includes spaces between list elements:
```suggestion
                exe_cmd = ['sh', '-c', 'cur=`pwd`; cd $AWS_FPGA_REPO_DIR ;\
```

Reply:
Thanks so much!! Just added the space!